Pyrado/pyrado/sampling/expose_sampler.py

```diff
@@ -26,13 +26,14 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+from abc import ABC
 from typing import List
 
 from pyrado.sampling.sampler import SamplerBase
 from pyrado.sampling.step_sequence import StepSequence
 
 
-class ExposedSampler:
+class ExposedSampler(ABC):
```

Comment:
Since Python typing does not allow intersections of types, this ensures that if some method requires an algorithm with `ExposedSampler` mixed in, it can indicate that just like this:
```python
def x(algo: ExposedSampler):
   # internal check
   assert all(isinstance(algo, c) for c in (Algorithm, ExposedSampler)) 
```